src/commonmark/en/content/user/system-settings.md

```diff
@@ -232,14 +232,20 @@
 <tbody>
 <tr class="odd">
 <td><p><strong>Select language</strong></p></td>
-<td><p>Sets the language for which you can then enter translations of the following settings:</p>
-<ul>
-<li><p><strong>Application introduction</strong></p></li>
-<li><p><strong>Application title</strong></p></li>
-<li><p><strong>Application notification</strong></p></li>
-<li><p><strong>Application left-side footer</strong></p></li>
-<li><p><strong>Application right-side footer</strong></p></li>
-</ul></td>
+<td>
```

Comment:
@HendrikThePendric can we switch to markdown style tables in new docs?

Reply:
@larshelge this is part of a (much) larger table. Are you suggesting I just update this section to use markdown formatting? Or should I reformat the entire table?

Reply:
Okay merging for now.